assets/scripts/admin/questionEditor.js

```diff
@@ -1514,7 +1514,9 @@ $(document).on('ready pjax:scriptcomplete', function () {
       });
       try {
         const [html, html2] = await Promise.all([generalSettingsPromise, advancedSettingsPromise]);
+        var currentGroup = $('#gid').children("option:selected").val();
```

Comment:
Can you use let or const instead, please?

Reply:
Done

Reply:
Sorry, I should have said: Use const (the value is never changed...)

Reply:
Well, I disagree with you on this one. From its conception is a variable, not a constant... I would keep with let... But if you insist, of course I will change it :)

Reply:
`const` should be used for variables that do not change. Compare with the `const` keyword in C++. Thanks, though.

Reply:
https://docs.microsoft.com/en-us/cpp/cpp/const-cpp?view=msvc-160